evaluation.html

```diff
@@ -0,0 +1,64 @@
+<DOCTYPE html>
+    <html>
+        <head><title></title>
+        
+        <link rel="stylesheet" type="text/css" href="css/evaluation.css">
+        </head>
+        
+        <body>
+            
+        <header>
+        <a id="homeLogo" href="login.html"><img src="assets/Logo.png"></a>   
+        <a class="pgSwitch , borderSwitch" href="evaluation.html">Evaluations</a> 
```

Comment:
Why the comma between the two classes? Also, are multiple classes allowed on HTML elements simultaneously?